build.gradle

```diff
@@ -431,8 +443,9 @@ def afterTestClosure = { descriptor, result ->
     }
 }
 
+import org.gradle.internal.serialize.PlaceholderException
 def checkConfigFailureClosure = { descriptor, result ->
-    if (result.exception instanceof org.gradle.messaging.remote.internal.PlaceholderException
+    if (result.exception instanceof PlaceholderException
```

Comment:
Do one of the following:
- Empty line between `import ...` and `def ...`
- Use fully qualified name as like before, only changing the package name as necessary